From 77bbaf26a936d37ab5352775454facbeecb91172 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Tue, 12 Mar 2019 23:39:42 -0700
Subject: [PATCH 0557/1851] drm: xlnx: dsi: Fix the drm property creation

commit f0005ecae68818935d05c782da377b0fcb8c4526 from
https://github.com/Xilinx/linux-xlnx.git

The DRM_MODE_PROP_PENDING flag is getting deprecated in 4.19 kernel.
So clear this flag to fix the drm property creation logic.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_dsi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_dsi.c b/drivers/gpu/drm/xlnx/xlnx_dsi.c
index 97878b33acd4..ce744c6c6178 100644
--- a/drivers/gpu/drm/xlnx/xlnx_dsi.c
+++ b/drivers/gpu/drm/xlnx/xlnx_dsi.c
@@ -532,7 +532,7 @@ static void xlnx_dsi_connector_create_property(struct drm_connector *connector)
 	struct drm_device *dev = connector->dev;
 	struct xlnx_dsi *dsi  = connector_to_dsi(connector);
 
-	dsi->eotp_prop = drm_property_create_bool(dev, 1, "eotp");
+	dsi->eotp_prop = drm_property_create_bool(dev, 0, "eotp");
 	dsi->video_mode_prop = drm_property_create_range(dev, 0, "video_mode",
 							 0, 2);
 	dsi->bllp_mode_prop = drm_property_create_bool(dev, 0, "bllp_mode");
-- 
2.31.1

